### DISCUSSION TITLE
No email sent, no logs when inviting via email, but UI says it was successful

### DISCUSSION THREAD

**Discussion by chase-replicated** [outline/outline#2605] [Self Hosting]
I've set up my SMTP settings from Mandrill in the .env. I was getting an error that I believe went away after setting the new flag "SMTP_SECURE" to false, but now there are no logs regarding mail. I set DEBUG=emails,mailer  but the only logs I see are 

Sequelize CLI [Node: 14.17.6, CLI: 6.2.0, ORM: 6.3.5]

Loaded configuration file "server/config/database.json".
Using environment "production".
No migrations were executed, database schema was already up to date.
Done in 0.80s.
yarn run v1.22.5
$ node ./build/server/index.js

Is your team enjoying Outline? Consider supporting future development by sponsoring the project:

https://github.com/sponsors/outline

**To Reproduce**
1. Set:
SMTP_HOST=company.com
SMTP_PORT=587
SMTP_USERNAME=value@domain.com
SMTP_PASSWORD=XXXXXXX
SMTP_FROM_EMAIL=noreply@domain.com
SMTP_REPLY_EMAIL=noreply@domain.com
SMTP_SECURE=true
DEBUG=emails,mailer
2. Run outline
3. Invite a new member
4. view stdout / system logs of container
5. View Mandrill statistics, no email received.

**Expected behavior**
See logs of some sort, have the mail go through and reach Mandrill

**Outline (please complete the following information):**
- Install: self hosted
- Version: latest(0.59.0)

**Desktop (please complete the following information):**
 - OS: Docker image outlinewiki/outline:latest
 - Browser Firefox
 - Version 92.0

**Comment by tommoor** [comment]
Are you sure you're running the latest docker image unedited? I ask because it looks like migrations are attempting to run which wouldn't be the case typically.

**Reply by chase-replicated** [reply]
We're running it on a kubernetes cluster, and I have an init container for the deployment set up to run migrations so every time we make changes and push to our kubernetes namespace, the migrations run again because it's as if it's being stood up again. 

The container pod is scheduled on an AMD64 node. Networking is working as all our teammates are able to login and edit.

**Reply by marianmurin** [reply]
@chase-replicated Have you solved your issue with emails? I have the same situation. I am running all Outline containers through Portainer.

**Reply by chase-replicated** [reply]
I was able to fix this by setting the environment variable `SMTP_TLS_CIPHERS` explicitly to an empty string like : `SMTP_TLS_CIPHERS=""`

**Comment by marianmurin** [comment]
I have tried it, but it does not help.
Here is my configurations:

### Docker compose config for email section

# EMAIL
- SMTP_HOST=${SMTP_HOST}
- SMTP_PORT=${SMTP_PORT}
- SMTP_USERNAME=${SMTP_USERNAME}
- SMTP_PASSWORD=${SMTP_PASSWORD}
- SMTP_FROM_EMAIL=${SMTP_FROM_EMAIL}
- SMTP_REPLY_EMAIL=${SMTP_REPLY_EMAIL}
- SMTP_TLS_CIPHERS=""
- SMTP_SECURE=${SMTP_SECURE}

### Variables

<img width="1358" alt="Variables" src="https://user-images.githubusercontent.com/14167315/147286363-ad0b655e-fe37-4a02-a813-6938244c5ae8.png">

### Logs from docker

<img width="1387" alt="Logs from docker" src="https://user-images.githubusercontent.com/14167315/147286376-4d0dbaee-92c0-4318-8575-ac6590132266.png">

**Comment by letran3691** [comment]
You try this. it's worked in my case.
SMTP_HOST="smtp.gmail.com"
SMTP_PORT="465"
SMTP_USERNAME="monitor-boot@gmail.com"
SMTP_PASSWORD="password email"
SMTP_FROM_EMAIL="monitor-boot@gmail.com"
SMTP_REPLY_EMAIL=""
SMTP_TLS_CIPHERS="TLSv1.2"
SMTP_SECURE="true"

**Reply by marianmurin** [reply]
Hi @letran3691 thank you for your tip, but it did not helped me.
Here is my logs from outline container:

{"label":"email","level":"info","message":"Sending email \"Marián Murín invited you to join Wiki iMPROVE’s knowledge base\" to support@improve.sk"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"backlinks processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"debouncer processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"imports processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"exports processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"notifications processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"revisions processing users.invite"}
{"name":"users.invite","modelId":null,"label":"processor","level":"info","message":"slack processing users.invite"}
{"error":"140573188848456:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:\n","stack":"Error: 140573188848456:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:\n","level":"error","message":"Error sending email to support@improve.sk"}
(node:70) UnhandledPromiseRejectionWarning: Error: 140573188848456:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
(Use `node --trace-warnings ...` to show where the warning was created)
(node:70) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

**Reply by tommoor** [reply]
Try `SMTP_TLS_CIPHERS=SSLv3`

https://stackoverflow.com/a/66317696/192065

**Reply by marianmurin** [reply]
I am happy that there are poeople who care about my problem and I appreciate that, but I have tried all of your tips and it dd not worked.
After searching some information I think it could be related with proxy server what i am using. But do not know how to debug it.

I am using NginxProxyManager docker as a proxy server.
